schemas/user.json

```diff
@@ -337,6 +337,11 @@
 					}
 				}
 			}
+		},
+		"initials": {
```

Comment:
Will the be the preferred avatar initials if those are present? Or will this always be the automatically generated ones?

Reply:
This will be the value returned by the _getDisplayInitials_ method introduced in this PR. Therefore when preferred avatar initials are available, those will be the value. Otherwise,  initials will be determined by the user's name.

After further consideration, I believe `displayInitials `would be a more appropriate name. The current name may imply that the initials are exclusively derived from the user's name, which could lead to confusion when preferred avatar initials are returned instead.